cvat-canvas3d/src/typescript/canvas3dView.ts

```diff
@@ -1875,4 +1877,11 @@ export class Canvas3dViewImpl implements Canvas3dView, Listener {
             front: this.views.front.renderer.domElement,
         };
     }
+
+    public destroy(): void {
+        this.dispatchEvent(new CustomEvent('canvas.destroy', {
```

Comment:
what is ``this`` here? Is it a DOM element? 
I am not sure, the current implementation makes sense, maybe just left empty API method that does nothing and implement it later when we will now what this API method should do in canvas3d

Reply:
Made empty destroy method in canvas3dModel